x-pack/libbeat/reader/reader.go

```diff
@@ -0,0 +1,48 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package reader
+
+import (
+	"bufio"
+	"bytes"
+	"compress/gzip"
+	"io"
+)
+
+// Moved from x-pack/filebeat/input/awss3/s3_objects.go
```

Comment:
I don't think comments like this should be here. This context is available through the git history and only pollutes the godoc.